data-prepper-expression/src/main/antlr/DataPrepperExpression.g4

```diff
@@ -240,6 +236,7 @@ NOT_IN_SET : SPACE 'not in' SPACE;
 AND : SPACE 'and' SPACE;
 OR : SPACE 'or' SPACE;
 NOT : 'not' SPACE;
+SUBTRACT : '-';
```

Comment:
Why is this called `SUBTRACT`? It seems to be a negation operator.

Reply:
@chenqi0805 and I had a short conversation about the name [here](https://github.com/opensearch-project/data-prepper/pull/1165#discussion_r824007751)